ts/screens/profile/LanguagesPreferencesScreen.tsx

```diff
@@ -126,6 +129,16 @@ const LanguagesPreferencesScreen = () => {
       setIsLoading(false);
       preferredLanguageSaveSuccessDispatch(selectedItem as Locales);
       setSelectedItem(selectedLanguage.current);
+      showModal(
+        <AlertModal
+          message={I18n.t("profile.main.pagoPaEnvironment.alertMessage")}
+        />
+      );
+      IOToast.success(
```

Comment:
Since we are in a functional component, wouldn't it be better to use the `useIOToast` hook instead?